Description: Remove value assignments to environment variable LD_RUN_PATH
 Value assignment to environment variable LD_RUN_PATH leads to setting RPATH in
 compiled binary/shared library.
 .
 And then Lintian reports about error (E: binary-or-shlib-defines-rpath).
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Author: Janos Guljas <janos@debian.org>
Forwarded: not-needed
Last-Update: 2016-02-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/plugins/rack/uwsgiplugin.py
+++ b/plugins/rack/uwsgiplugin.py
@@ -46,7 +46,6 @@
 
 if has_shared == 'yes':
     LDFLAGS.append('-L' + libpath )
-    os.environ['LD_RUN_PATH'] = libpath
     LIBS.append(os.popen(RUBYPATH + " -e \"require 'rbconfig';print '-l' + %s::CONFIG['RUBY_SO_NAME']\"" % rbconfig).read().rstrip())
 else:
     rubylibdir = os.popen(RUBYPATH + " -e \"require 'rbconfig';print RbConfig::CONFIG['rubylibdir']\"").read().rstrip()
--- a/plugins/python/uwsgiplugin.py
+++ b/plugins/python/uwsgiplugin.py
@@ -60,9 +60,6 @@
             libdir = "%s/lib" % sysconfig.PREFIX
 
         LDFLAGS.append("-L%s" % libdir)
-        LDFLAGS.append("-Wl,-rpath,%s" % libdir)
-
-        os.environ['LD_RUN_PATH'] = "%s" % libdir
 
         LIBS.append('-lpython%s' % get_python_version())
 else:
